meinberlin/apps/livequestions/assets/Filters.jsx

```diff
@@ -73,6 +74,7 @@ export default class Filter extends React.Component {
                   checked={this.props.displayNotHiddenOnly}
                   onChange={this.props.toggledisplayNotHiddenOnly} // eslint-disable-line react/jsx-handler-names
                 />
+                <span className="visually-hidden">{displayNotHiddenText}</span>
```

Comment:
added here as sr will read out content of label by default so seemed better then aria-label